### ISSUE TITLE
How to work on tests closer to imelementations?

### ISSUE THREAD

**Issue by galargh** [ipfs/gateway-conformance#86]
Does write a test in the implementation, run it for a while, contribute it back to gateway conformance work with our current setup? If so, let's describe how to do it. Otherwise, what would it take to get us there? I think it could lower the bar to start writing gateway conformance tests.

**Comment by laurentsenta** [comment]
Definitely,
I'd love to see the toolings and fixtures reused, I think the stuff we wrote for car checks, tar checks, ipns checks, etc. might be useful in a toolbox for unit-testing.

Regarding the test suite itself, would documenting how to setup a fork and run a single, custom spec, fullfill your expectations?

**Comment by galargh** [comment]
Yes, I was thinking of documenting how to run a single custom spec without needing a fork. Let's talk more about this next week. Might be a fun little exercise to pair on :)
